HolyshiftService/JwtService.cs

```diff
@@ -0,0 +1,72 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+public class JwtService
+{
+    private readonly IOptions<AuthConfig> _authConfig;
+    private readonly byte[] _keyBytes = new byte[32];
+
+    public JwtService(IOptions<AuthConfig> authConfig)
+    {
+        _authConfig = authConfig;
+        _keyBytes = Convert.FromBase64String(_authConfig.Value.Key);
+    }
+
+    public string GenerateJwtToken(UserDbModel user)
+    {
+        var config = _authConfig.Value;
+        var securityKey = new SymmetricSecurityKey(_keyBytes);
+        var credentials = new SigningCredentials(securityKey, SecurityAlgorithms.HmacSha256);
+        var claims = new[]
+        {
+            new Claim(JwtRegisteredClaimNames.Sub, user.Id.ToString()),
+            new Claim(JwtRegisteredClaimNames.Email, user.Email),
+            new Claim(JwtRegisteredClaimNames.Jti, Guid.NewGuid().ToString()),
```

Comment:
"JwtRegisteredClaimNames.Jti"  its used to provide unique identifier for the token, can be used to prevent the replay of tokens and to ensure that a token hasn't been used before , but I can remove it if you see it unnecessary